src/analysis/connectivity/score_inputs.sql

```diff
@@ -0,0 +1,2636 @@
+----------------------------------------
+-- INPUTS
+-- location: neighborhood
+----------------------------------------
+DROP TABLE IF EXISTS generated.neighborhood_score_inputs;
+
+CREATE TABLE generated.neighborhood_score_inputs (
+    id SERIAL PRIMARY KEY,
+    category TEXT,
+    score_name TEXT,
+    score NUMERIC(16,4),
+    notes TEXT,
+    human_explanation TEXT,
+    use_pop BOOLEAN,
+    use_emp BOOLEAN,
+    use_k12 BOOLEAN,
+    use_tech BOOLEAN,
+    use_univ BOOLEAN,
+    use_doctor BOOLEAN,
+    use_dentist BOOLEAN,
+    use_hospital BOOLEAN,
+    use_pharmacy BOOLEAN,
+    use_retail BOOLEAN,
+    use_grocery BOOLEAN,
+    use_social_svcs BOOLEAN,
+    use_parks BOOLEAN,
+    use_trails BOOLEAN,
+    use_comm_ctrs BOOLEAN,
+    use_transit BOOLEAN
+);
+
+
+-------------------------------------
```

Comment:
I can see the logic in that. I've generally tried to followed a pattern of naming each script for the table which it creates (which as you note fails for the access_*.sql scripts).

The one concern I have with this approach is that the final scoring requires that we combine scores from several related categories. For instance, the core services score is a composite of doctors, dentists, hospitals, etc. If we break out this scoring into each access_*.sql file, we'll still end up having to do some calculations within overall_scores.sql.

I'll mull this as I make these other changes.